web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -199,8 +199,8 @@ export namespace TaskItem {
           class="md-taskitem ${classMap({ selected: this.selected })}"
           tabindex="0"
           aria-selected="${this.selected}"
-          @click=${(e: MouseEvent) => this.handleClick(e)}
-          @keydown=${(e: KeyboardEvent) => this.handleKeyDown(e)}
+          @click=${this.handleClick}
```

Comment:
I dont think this changes have any impact to performance. Please revert this kind of changes. @prakhar-tiwari Right?

Reply:
I think it is better to not use arrow function here. As using arrow function will separate function whenever it gets called

Reply:
Removed these changes